gulpfile.js

```diff
@@ -112,7 +112,9 @@ gulp.task('coverage', ['lint', 'build'], function () {
     'use strict';
     
     return gulp.src(paths.sourceFiles)
-        .pipe(istanbul())
+        .pipe(istanbul({
+            includeUntested: true
```

Comment:
You should omit this change. It tends to break the build. We're going to make a separate branch to address this issue.